assets/questions/strobogrammatic.js

```diff
@@ -0,0 +1,78 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for strobogrammatic number.
+ */
+
+globalData.questions['strobogrammatic'] = {  // eslint-disable-line dot-notation
+  title: 'strobogrammatic',
+  starterCode: {
+    python:
+`def strobogrammatic(length):
+    return []
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def firstDigitZero(cls, length):
+        result = [["00","11","69","88","96"]]
+        return result
+
+    @classmethod
+    def forgetZero(cls, word):
```

Comment:
Why does this take "word" as an argument? And should it always return the same result, regardless of the input?